### PR TITLE
assessment: .multi levels can have "layout 'tight'" for more compact layout

### PR DESCRIPTION
**Pull request by breville**
screenshot 2016-03-28 17 30 42
screenshot 2016-03-28 17 30 49

### CODE REVIEW

File: dashboard/app/views/levels/_single_multi.html.haml [hunk 1]
```diff
@@ -44,7 +45,8 @@
       %form#voteform{onsubmit: 'return false;'}
         - data['answers'].each_with_index do |answer, i|
           - correct = answer['correct']
-          %span.answerbutton.btn{style: "height: #{height}", correct: "#{correct}", index: "#{i}"}
+          - answer_class = data['layout'] == "tight" ? "answerbutton answerbutton-tight" : "answerbutton answerbutton-default btn"
```
Comment (laurelfan) [inline dashboard/app/views/levels/_single_multi.html.haml]: answerbutton-tight doesn't need btn?

**Comment by laurelfan** [comment]
is the tight layout exclusively for long assessments? If so, would it make sense for all multis that are in long assessments to have tight layout (and none of them that are outside of long assessments)?

File: dashboard/app/views/levels/_single_multi.html.haml (hunk 1 above)
Comment (breville) [inline dashboard/app/views/levels/_single_multi.html.haml]: correct; with btn it gets the grey button visual which we don't want here.

**Comment by breville** [comment]
Fair idea: I've now made it so that LevelGroup uses tight layout by default, though it's also possible to specify "layout: tight" in a regular .multi in case somebody wants to force its use there.  I expect this situation might be fluid until we see what the curriculum team prefers.

**Pull request merged**